### DISCUSSION TITLE
prefect support MS SQL Server as Database

### DISCUSSION THREAD

**Discussion by keshavksingh** [PrefectHQ/prefect#8843] [Ideas] [answered]
I was trying to setup the prefect latest server, and it appears the supported backend is PostgreSQL or  SqlLite. I wasn't able to get it working with Azure MS SQL db instance as a backend. Would appreciate inputs/help.

**Comment by gabcoyne** [comment, marked as answer]
Hey @keshavksingh we don't support MSSQL as a backend. I'd use an Azure Postsgres instance if you are using azure. https://azure.microsoft.com/en-us/products/postgresql
